pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_pantheon-user-interface.adoc

```diff
@@ -0,0 +1,28 @@
+[id='pantheon-user-interface']
+= Pantheon user interface
+
+
+The user interface is organized with a main work area, an upper toolbar with a `Login` button and a show/hide button for displaying the sidebar, and a tabbed sidebar with the following tabs:
+
+* `Modules` - Use this tab to create new document modules, search for, edit, or delete modules.
+** `Search` - Provides advanced search options, including keyword search, search by product name, product version, document module  type (procedure, concept, reference) and upload date.
+*** Select a module check box to delete the selected module.
+*** Select a module name to view the module status and details, and publish, unpublish, preview or view the module in the customer portal.
```

Comment:
I don't know how granular you'd like to get, but it might be worth calling out that you can search without being authenticated, but the Module Name link will take you straight to a doc preview in that case, not the module details screen. (Some of the other tabs below might be hidden depending on whether you're logged in and what role your user has).

Up to you whether such details should make it into the docs, I just wanted to make sure you were aware of the difference in functionality.

Reply:
Thanks for pointing it out Ben, yes, I realize that there are currently at least 3 levels of permissions - unauthenticated or not-logged in, logged in User, and logged in Administrator. 
I was thinking of mentioning the different permissions somewhere, and linking to that description in the Log in chapter, and I discussed it with Andrew, but he thinks we can wait with that topic.

Reply:
Fair enough, just wanted to make sure you were aware!